### PR TITLE
DotCMS/core#27956 Implement styles for prime-ng Image component and apply to the AI Image dialog. 

### LINKED ISSUE #27956: Implement styles for prime-ng Image component and apply to the AI Image dialog.
### Parent Issue
#27632

### User Story

Add the prime ng image component with the. correct styles in the Ai image dialog. 

### Acceptance Criteria

- Add the theme styles for the `p-image` component
- Integrate the syles in the AI image pop up 
- Create the storybook to `p-image`

### dotCMS Version
master

### Proposed Objective
_to_define_

### Proposed Priority
_to_define_

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.
_to_define_

### Assumptions & Initiation Needs
_to_define_

### Quality Assurance Notes & Workarounds
_to_define_

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Add `p-image` component to the repo. 
* Add storybook. 
* Integrate it to the Ai Image pop-up

https://github.com/dotCMS/core/assets/31667212/fba788b3-4328-4ecb-982d-c9b69aa89032

### CODE REVIEW

File: core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss [hunk 1]
```diff
@@ -1 +1,50 @@
...
+.p-image-toolbar {
+    padding: $spacing-3;
+}
+
+.p-image-action.p-link {
+    color: $white;
+    background-color: transparent;
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
```
Comment (oidacra) [inline core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss:29]: Does exist a css-var for the border radius?

File: core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss [hunk 2]
```diff
@@ -1 +1,50 @@
...
+    transition: background-color $basic-speed, color $basic-speed, box-shadow $basic-speed;
+    margin-right: 0.5rem;
```
Comment (oidacra) [inline core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss:31]: `$spacing-1`?

File: core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss [hunk 3]
```diff
@@ -1 +1,50 @@
...
+.p-component-overlay.p-image-mask {
+    background-color: $black;
+}
+
+.p-image-preview-indicator {
+    background-color: transparent;
+    color: $white;
+
+    .pi-window-maximize {
+        font-size: 2rem;
```
Comment (oidacra) [inline core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss:12]: Works `$font-size-xxl` for you?

Comment (hmoreras) [reply core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss:12]: yes, not big difference.

File: core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss (hunk 1 above)
Comment (hmoreras) [reply core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss:29]: no but just create `$border-radius-circular`

**Review by zJaaal — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/stories/primeng/media/Image.stories.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            description: {
+                component: 'Displays an image with preview and tranformation options.'
+            }
+        }
+    }
+} as Meta;
+
+const Template = `<p-image [src]="src" [alt]="alt" [preview]="preview"></p-image>`;
+
+export const Main: Story = (args) => {
```
Comment (fmontes) [inline core-web/apps/dotcms-ui/src/stories/primeng/media/Image.stories.ts:22]: `Story` is deprecated, can we check what's the new thing? so we don't use deprecated things in new stories.

File: core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss [hunk 5]
```diff
@@ -1 +1,50 @@
...
+    transition: background-color $basic-speed, color $basic-speed, box-shadow $basic-speed;
+    margin-right: $spacing-1;
+
+    &:last-child {
+        margin-right: 0;
+    }
+
+    &:hover {
+        color: $white;
+        background: rgba(255, 255, 255, 0.2);
```
Comment (fmontes) [inline core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss:39]:
```suggestion
        background: $color-palette-white-op-20;
```

**Review by fmontes — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/stories/primeng/media/Image.stories.ts (hunk 4 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/stories/primeng/media/Image.stories.ts]: moved to `StoryFn`

**Pull request merged**